client/extensions/woocommerce/state/wc-api/endpoints/products/index.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+export default endpointData => {
+	// TODO: Remove eslint ignore after requirements is implemented.
+	// eslint-disable-next-line no-unused-vars
+	function getSingle( productId, requirements ) {
```

Comment:
Just so i'm following along correctly, `requirements` is where things like freshness would be included?

Reply:
Precisely. The `requirements` options define this component's data requirements around the data returned by the selector. In cases of more complex selectors that might merge data from more than one endpoint, for example, the requirements would be applied to all data used by that selector. It's the cleanest way I can think of to track how data is needed to be consumed.